Directory.Build.props

```
@@ -5,8 +5,8 @@
     <LangVersion>latest</LangVersion>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageIcon>logo.64x64.png</PackageIcon>
-    <Copyright>Pekka Heikura</Copyright>
-    <Authors>Pekka Heikura</Authors>
+    <Copyright>Copyright (c) 2022 Shane Krueger, Pekka Heikura, et al. All rights reserved.</Copyright>
```

Since it came to a change in this property, I propose to mention top 3 contributors excluding bot. Also ping @pekkah to approve these changes.

---

```suggestion
    <Copyright>Copyright (c) 2015-2022 Shane Krueger, Pekka Heikura, Ivan Maximov, et al. All rights reserved.</Copyright>
```

This now matches the license.md file